src/platform/plugins/shared/chart_expressions/expression_xy/public/helpers/annotations.tsx

```diff
@@ -221,3 +231,24 @@ export function Marker({
   }
   return null;
 }
+
+const styles = {
+  xyAnnotationNumberIcon: ({ euiTheme }: UseEuiTheme) =>
+    css({
+      borderRadius: euiTheme.size.base,
+      minWidth: euiTheme.size.base,
+      height: euiTheme.size.base,
+      backgroundColor: 'currentColor',
+    }),
+  xyAnnotationNumberIconText: ({ euiTheme }: UseEuiTheme) =>
+    css({
+      fontWeight: euiTheme.font.weight.medium,
+      letterSpacing: '-.5px',
+    }),
+  xyAnnotationIconRotate90: css({
```

Comment:
As you have done in the `src/platform/plugins/shared/chart_expressions/expression_xy/public/components/annotations.tsx` file you can probably remove the `xyAnnotation` prefix on each style